compiler/ballerina-lang/src/main/java/io/ballerina/projects/BalaWriter.java

```diff
@@ -376,4 +393,26 @@ private String convertPathSeperator(Path file) {
             return file.toString();
         }
     }
+
+    /**
+     * Check package is a `ballerinai` package or `ballerina/observe` package.
+     *
+     * @param context package context
+     * @return package is a `ballerinai` package or `ballerina/observe` package
+     */
+    private boolean isBallerinaiOrBallerinaObservePackage(PackageContext context) {
+        return isBallerinaiOrBallerinaObservePackage(context.packageOrg(), context.packageName());
+    }
+
+    /**
+     * Check package is a `ballerinai` package or `ballerina/observe` package.
+     *
+     * @param packageOrg  package org
+     * @param packageName package name
+     * @return package is a `ballerinai` package or `ballerina/observe` package
+     */
+    private boolean isBallerinaiOrBallerinaObservePackage(PackageOrg packageOrg, PackageName packageName) {
+        return BALLERINA_INTERNAL_ORG.getValue().equals(packageOrg.value()) ||
+                (packageOrg.isBallerinaOrg() && packageName.value().equals("observe"));
```

Comment:
We have a constant to capture "observe" package name. i.e `Names.OBSERVE.value`

Reply:
Ohh I missed it, Will add now.